cmake/projects/Lua/hunter.cmake

```diff
@@ -8,6 +8,17 @@ include(hunter_pick_scheme)
 include(hunter_cacheable)
 include(hunter_configuration_types)
 
+hunter_add_version(
+    PACKAGE_NAME
+    Lua
+    VERSION
+    "5.3.5"
```

Comment:
Can we use 5.3.5-p0 for consistency

Reply:
I am not sure there is a consistency but I would like to offer a suggestion. I think it might be nice for all the first releases to not have a p0 number, with the expectation that there will be no patch and that the next version is the next release. However if a patch is release then we take on the -pn release starting at n=1 as the first release is p0. Let me know if you like this or not and I will change it accordingly.

Reply:
If you look at the default versions CMake file, a lot of packages have been using -p0. Maybe not all, but it’s definitely a convention. Anyways, I don’t really care, but we should keep the release tag and CMake version consistent at the least